practice-two/src/index.html

```diff
@@ -349,7 +349,120 @@ <h2 class="heading-section">Start Selling Instantly</h2>
+          <ul class="footer-section__list">
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Community</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Forum</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Distributors</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Privacy Policy</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Terms of Use</a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <h3 class="footer-section__title">Customers</h3>
+          <ul class="footer-section__list">
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Documentation</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Tools</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Downloads</a>
+            </li>
+            <li>
+              <a href="javascript:void(0)" class="text-normal footer-section__link">Developer Center</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-section__contact">
+          <div class="footer-section__social">
+            <a href="javascript:void(0)" class="footer-section__social-link">
+              <svg width="24" height="24">
+                <use xlink:href="assets/images/icon-footer.svg#instagram"></use>
+              </svg>
+            </a>
+            <a href="javascript:void(0)" class="footer-section__social-link">
+              <svg width="35" height="22">
+                <use xlink:href="assets/images/icon-footer.svg#google"></use>
+              </svg>
+            </a>
+            <a href="javascript:void(0)" class="footer-section__social-link">
+              <svg width="30" height="24">
+                <use xlink:href="assets/images/icon-footer.svg#twitter"></use>
+              </svg>
+            </a>
+            <a href="javascript:void(0)" class="footer-section__social-link">
+              <svg width="24" height="24">
+                <use xlink:href="assets/images/icon-footer.svg#facebook"></use>
+              </svg>
+            </a>
+            <a href="javascript:void(0)" class="footer-section__social-link">
+              <svg width="24" height="24">
+                <use xlink:href="assets/images/icon-footer.svg#camera"></use>
+              </svg>
+            </a>
+          </div>
+          <div class="text-center footer-section__info">
+            <p class="text-normal footer-section__info-name">© 2016 Hofmann UI Kit</p>
```

Comment:
Should use HTML Symbol Entities instead of copy/paste the special character. You can refer this article https://www.w3schools.com/html/html_symbols.asp